ossec_security/assets/logs/ossec-security.yaml

```
@@ -0,0 +1,708 @@
+              "Mozilla/5.0 (Android 7.1.1; Mobile; rv:64.0) Gecko/64.0
+              Firefox/64.0" "-"
+            - >
+              10.10.10.10 - - [22/Jan/2019:03:57:14 +0330] "GET
+              /filter/p10280%2Cv1%7C%D8%B7%D9%88%D8%B3%DB%8C%20%D9%86%D9%82%D8%B1%D9%87%20%D8%A7%DB%8C%2Cv1%7C%D8%B3%D9%81%DB%8C%D8%AF.?o=v1
+              HTTP/1.1" 302 0 "-" "Mozilla/5.0 (compatible; AhrefsBot/6.1;
+              +http://ahrefs.com/robot/)" "-"
+            - >
+              10.10.10.10 - - [22/Jan/2019:03:56:29 +0330] "GET
+              /image/45443/productModel/150x150 HTTP/1.1" 200 2891 "-"
+              "Mozilla/5.0 (compatible; bingbot/2.0;
+              +http://www.bing.com/bingbot.htm)" "-"
+            - 10.10.10.10 - - [22/Jan/2019:03:56:29 +0330] "GET
+              /image/45443/productModel/150x150 HTTP/1.1" 200 2891
+            - 10.10.10.10 - - [22/Jun/2007:09:02:39 -0300] "POST
+              /dcid/wp-trackback.php?p=9 HTTP/1.0" 401 464 "" "Mozilla/4.0
+              (compatible; MSIE 6.0; Windows NT 5.1; SV1)"
+          grok:
+            supportRules: |-
+              TILL_DOUBLE_QUOTE_PARSE_RULE %{regex("[^\"]*"):}
+              TILL_CLOSING_BRACKET_PARSE_RULE %{regex("[^]]*"):}
+            matchRules: web_access_parsing_rule %{ip:src_ip} %{notSpace} %{notSpace}
+              \[%{TILL_CLOSING_BRACKET_PARSE_RULE}\] "%{word:http.method}
+              %{notSpace:request} %{notSpace:http.request_id}"
+              %{integer:http.status_code} %{integer:size}(
+              "%{TILL_DOUBLE_QUOTE_PARSE_RULE:referer:nullIf("-")}")?(
+              "%{data:http.useragent}")?( %{quotedString:})?
+        - type: user-agent-parser
+          name: Define `http.useragent` as `http.useragent_details`
+          enabled: true
+          sources:
+            - http.useragent
+          target: http.useragent_details
+          encoded: false
+          combineVersionDetails: false
+    - type: pipeline
+      name: Processing of System Logs
+      enabled: true
+      filter:
+        query: "@rule_id:(501 OR 503 OR 504 OR 591) "
+      processors:
+        - type: grok-parser
+          name: Parsing of OSSEC System logs
+          enabled: true
+          source: log
+          samples:
+            - "ossec: Agent started: 'dummy_agent->10.10.10.10'."
+            - "ossec: Agent disconnected: 'dummy_agent-10.10.10.10'."
+            - "ossec: File rotated (inode changed):
+              '/var/log/apache2/access.log'."
+            - "ossec: File size reduced (inode remained):
+              '/var/log/ntrace/ntrace.log'."
+          grok:
+            supportRules: ""
+            matchRules: >-
+              ossec_agent_parsing_rule ossec: Agent %{word:action}:
+              '%{notSpace:agent-name}-(>)?%{notSpace:agent-ip}'.
+
+              log_file_change_parsing_rule ossec: File %{regex("[A-Za-z ]*"):action} (%{regex("[^)]*"):})\): '%{notSpace:file}'.
+    - type: pipeline
+      name: Processing of Low-Disk Space Logs
```

```suggestion
      name: Processing of Low Disk Space Logs
```
Because the disk space is low, not just the disk

---

Updated as per suggestion